<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xiaoqu_Manage</class>
 <widget class="QMainWindow" name="xiaoqu_Manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">#frame_3{border:2px solid rgb(121,121,121)}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QFrame" name="frame">
             <property name="styleSheet">
              <string notr="true">#frame{border:2px solid rgb(121,121,121)}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>小 区  名 称：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_xqname">
                    <property name="minimumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>负责人姓名：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_name">
                    <property name="minimumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>性    别：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_sex">
                    <property name="minimumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>联 系  方 式：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_tel">
                    <property name="minimumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>登   录   名：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_username">
                    <property name="minimumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>登 陆  角 色：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_play">
                    <property name="minimumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>客 服  电 话：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_kftel">
                    <property name="minimumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>14</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>最后操作时间：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_lasttime">
                    <property name="minimumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>311</width>
                      <height>51</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>宋体</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="styleSheet">
              <string notr="true">#frame_2{border:2px solid rgb(121,121,121)}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QLabel" name="label_8">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>111</width>
                    <height>51</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>111</width>
                    <height>51</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>小区介绍</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEdit">
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_update">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>51</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>提交修改</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
